src/bijectors/corr.jl

```diff
@@ -0,0 +1,109 @@
+# See stan doc for parametrization method:
+# https://mc-stan.org/docs/2_23/reference-manual/correlation-matrix-transform-section.html
+# (7/30/2020) their "manageable expression" is wrong...
+
+struct CorrBijector <: Bijector{2} end
+
+(b::CorrBijector)(X::AbstractMatrix{<:Real}) = link_lkj(X)
+(b::CorrBijector)(X::AbstractArray{<:AbstractMatrix{<:Real}}) = map(b, X)
+
+(ib::Inverse{<:CorrBijector})(Y::AbstractMatrix{<:Real}) = inv_link_lkj(Y)
+(ib::Inverse{<:CorrBijector})(Y::AbstractArray{<:AbstractMatrix{<:Real}}) = map(ib, Y)
+
+
+logabsdetjac(::Inverse{CorrBijector}, y::AbstractMatrix{<:Real}) = log_abs_det_jac_lkj(y)
+function logabsdetjac(b::CorrBijector, X::AbstractMatrix{<:Real})
+    
+    return -log_abs_det_jac_lkj(b(X)) # It may be more efficient if we can use un-contraint value to prevent call of b
+end
+logabsdetjac(b::CorrBijector, X::AbstractArray{<:AbstractMatrix{<:Real}}) = mapvcat(X) do x
+    logabsdetjac(b, x)
+end
+
+
+function log_abs_det_jac_lkj(y)
+    # it's defined on inverse mapping
+    K = size(y, 1)
+    
+    z = tanh.(y)
```

Comment:
I guess this should just be computed inside of the loop below, the allocations here seem unnecessary.

Reply:
`z` is used in three reverse AD backends. Though it's possible to optimize ForwardDiff specifically.

Reply:
Removed the allocations.